announcements/2020-Week-32/aws.md

```diff
@@ -0,0 +1,16 @@
+# :zap: This Week in AWS Releases (August 7, 2020):zap:
+
+[AWS v11.5.1](https://github.com/giantswarm/releases/tree/master/aws/v11.5.1) fixes a problem with the calculation of the maximum number of Pods per node and improves resource cleanup after cluster deletion.
+
+[AWS v12.1.0](https://github.com/giantswarm/releases/tree/master/aws/v12.1.0) upgrades Cert Manager. (Discussion with SE prior to ugprade is recommended.)
+
+**Optional Apps Releases**  
```

Comment:
@cokiengchiara shall we move optional apps announcements to a separate file?

Assumption is that most if not all optional apps are provider agnostic, and that it would reduce announcement noise (optional apps announcement duplication) for customers which have clusters on multiple providers (e.g. adidas on kvm and aws), dvag (on azure and kvm)

Reply:
thanks for raising this! hmm. so in this file, link to an "optional apps releases" announcement file?

Reply:
Separate apps.md file in every week directory (whenever there are some optional app releases), sent to all customers once. Link in provider specific files (like aws.md) is optional. Whatever is currently used to send provider specific announcements, would need to be modified to send apps.md too.

Reply:
Had a meeting with @cokiengchiara and @MartaD - decision was to do it next week, likely with updated docs on the process.